docs/modules/ROOT/pages/how-tos/metadata/index.adoc

```diff
@@ -1 +1,56 @@
-= Inspecting provenance and attestations
\ No newline at end of file
+= Inspecting provenance and attestations
+
+We are committed to providing exceptional security with {ProductName} by hardening the build platform and providing transparency into the build process and artifact's composition. By hardening the build platform, we enable multiple builds to run on a shared platform while preventing tampering from other running builds (link:https://slsa.dev/spec/v1.1/threats-overview[threats to the build process]). By providing transparency, we enable xref:/advanced-how-tos/managing-compliance-with-ec.adoc[policy evaluation] and auditability on the actual build process (link:https://slsa.dev/spec/v1.1/threats-overview[threats to the package selection]). This commitment means that {ProductName} conforms to the link:https://slsa.dev/spec/v1.0/levels#build-l3[SLSA Build Level 3] requirements.
+
+Don't take our word on the transparency -- you can download and inspect these artifacts yourself! {ProductName} generates and pushes build metadata as OCI artifacts to the container registry alongside your software's build using the link:https://github.com/oras-project/artifacts-spec/blob/main/manifest-referrers-api.md[referrers API].
+
+== Discovering the associated metadata
+
+.Prerequisites
+
+* Install the link:https://docs.sigstore.dev/cosign/installation/[Cosign] CLI tool.
+
+* Install the link:https://stedolan.github.io/jq/download/[jq] CLI tool.
+
+* xref:/getting-started/cli.adoc[Login] to {ProductName}.
+
+.Procedure
+
+In the CLI, complete the following steps to discover the associated metadata a component:
```

Comment:
"the associated metadata *for* a component"

Reply:
```suggestion
In the CLI, complete the following steps to discover the associated metadata for a component:
```